### ISSUE TITLE
[BUG] formulas state fails on Windows

### ISSUE THREAD

**Issue by noelmcloughlin** [saltstack-formulas/salt-formula#485]
### Pillar / config used

---

### Describe the bug

I was running salt-formula/init.sls and there are many state failures on Windows which need investigation. 

Log attached.

**Comment by myii** [comment]
@noelmcloughlin That's a bit strange since we're successfully testing on Windows here:

* https://github.com/saltstack-formulas/salt-formula/runs/1028772636?check_suite_focus=true

Perhaps the `salt-minion` isn't installed properly.  You can compare with this:

https://github.com/saltstack-formulas/salt-formula/blob/2dcd0c80efbd0acd0ca832a3435b7f3dc21788eb/kitchen.github.yml#L13-L20

**Comment by noelmcloughlin** [comment]
The suite runs `salt.minion` which probably works okay. 
I'm running `salt.init` which calls `salt.formulas` and that state is giving failures.

**Comment by noelmcloughlin** [comment]
Looks like  `parallel: True` does not work on Windows!
```
foo_app:
  git.latest:
    - name: https://github.com/saltstack-formulas/salt-formula.git
    - target: /tmp/bob
    - parallel: False  # True causes failed state
```

**Comment by noelmcloughlin** [comment]
This is bad logic too.  `user` and `group` are looped/rendered multiple times.  
https://github.com/saltstack-formulas/salt-formula/blob/2dcd0c80efbd0acd0ca832a3435b7f3dc21788eb/salt/formulas.sls#L38-L44

**Comment by noelmcloughlin** [comment]
After testing #486 there is one outstanding error on Windows. Not sure what error is about. There is a `master` branch
```
          ID: c:\salt\srv\salt\namespaces\saltstack-formulas/salt-formula_base
    Function: git.latest
        Name: https://github.com/saltstack-formulas/salt-formula.git
      Result: False
     Comment: Unable to get position of local branch 'master': fatal: ambiguous argument 'master^commit': unknown revision or path not in the working tree.
              Use '--' to separate paths from revisions, like this:
              'git <command> [<revision>...] -- [<file>...]'
```

**Comment by noelmcloughlin** [comment]
Actually ignore that error (I'm testing a PR branch of salt-formula and `salt.formulas` state is running `git.latest` state for salt-formula generating the error).   So all issues resolved in #486
